models/Tutorial.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const tutorialSchema = new Schema({
+  id: { type: String, required: true },
```

Comment:
Why set an id here? You will end up with two id fields in the database, because an _id field will automatically be created.